lineapy/api/api.py

```diff
@@ -212,24 +212,19 @@ def _try_delete_pickle_file(pickled_path: Path) -> None:
             raise KeyError(f"Pickle not found at {pickled_path}")
 
 
-def _try_write_to_db(value: object) -> Path:
+def _pickle_name(node_id: LineaID, execution_id: LineaID) -> str:
+    return f"{hash(node_id)}-{hash(execution_id)}-pickle"
```

Comment:
the name can become quite long with two hashes of big text ids. i recommend combining node_id and execution_id together as a string then hashing it. Also add _both_ a prefix and a suffix. because hashes can be negatives, it would look weird if filenames start with dashes.